stories/screen.js

```diff
@@ -66,6 +67,7 @@ storiesOf('Screens', module)
   .add('Loader - First Time', () => <Loader />)
   .add('Seed', () => <Seed store={store} />)
   .add('Seed Success', () => <SeedSuccess />)
+  .add('New Address', () => <NewAddress store={store} invoice={invoice} />)
```

Comment:
This is just nitpicking, but I reordered the screens so they are in line with the user journey. This way it's easier to follow in storybook.